lib/components/activity-indicator/s-activity-indicator.a11y.test.ts

```diff
@@ -0,0 +1,21 @@
+import { runComponentTests } from "../../test/test-utils";
+import "../../index";
+
+describe("s-activity-indicator", () => {
```

Comment:
Is there a specific reason why for the test files we kept the `s-` prefix? For consistency I would prefer to drop the `s-` entirely when naming files.

Reply:
No specific reason, just an oversight. I'll drop the prefix.